projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.ts

```diff
@@ -60,38 +55,23 @@ export class ResourceViewComponent implements OnInit, OnChanges {
       (response: ReadResource) => {
```

Comment:
the param docstring of getResource says resource instead of iri.

Reply:
what do you mean?

Reply:
the documentation string for the method

Reply:
ok, the related line to your comment misled me...

Reply:
resolved in 1972f45